### PR TITLE
Warn about ambiguous imports

### PR DESCRIPTION
**Pull request by exonian**
I tried to make this work with allies, and I'd like it to work on all selections rather than just units (although that's less important in practice) but I think I'm gonna call time on both of those. I'll be happy if you want to take those on, but I wanted to get the basic version over the line myself.

Apologies for anything grossly unidiomatic 😁 
ambiguous

### CODE REVIEW

File: src/utils/import/ambiguousSelections.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { TImportError } from 'types/import'
+import { createAmbiguityWarning } from 'utils/import/warnings'
+
+/**
+ * Mutates the errors array if it finds a suitable match in the allegiance abilities
```
Comment (exonian) [inline src/utils/import/ambiguousSelections.ts]: Oops missed this in the copy-paste fun

File: src/tests/azyr/azyrImport.test.ts [hunk 2]
```diff
@@ -544,6 +545,18 @@ describe('handleAzyrPages', () => {
     ])
   })
 
+  it('handles Stormcast5, which has a vague unit name', () => {
```
Comment (exonian) [inline src/tests/azyr/azyrImport.test.ts]: This test can probably go, actually, it's not really giving us anything now the other one exists

File: src/utils/import/options.ts [hunk 3]
```diff
@@ -240,8 +240,19 @@ export const importFactionNameMap = {
 
 export type TNameMap = { [key: string]: string }
 
+// A list of unit names that Azyr uses for multiple warscrolls
+// (eg 'Lord-Arcanum on Celestial Dracoline' is one of a number of units just called 'Lord-Arcanum')
```
Comment (daviseford) [inline src/utils/import/options.ts]: Great comment, everything fell into place in my head once I read this.

**Comment by exonian** [comment]
new-warnings

**Pull request merged**